features/manage_forms.feature

```diff
@@ -101,6 +101,28 @@ Feature: Manage Forms
     Then I should see "Test Form"
     And I should see "What is your gender?"
 
+  Scenario: Create New Form from List and Create a Question using New Question Modal
+    Given I have a Response Set with the name "Gender Full"
+    And I have a Question with the content "What is your gender?" and the type "MC"
+    And I am logged in as test_author@gmail.com
+    When I go to the list of Forms
+    And I click on the "New Form" link
+    And I fill in the "name" field with "Test Form"
+    And I fill in the "controlNumber" field with "1234-1234"
+    And I fill in the "description" field with "Form description"
+    And I click on the "Add New Question" button
+    And I fill in the "Question" field with "What is your favorite color?"
+    And I fill in the "question_description" field with "This is a description"
+    And I click on the "Add Question" button
+    And I click on the button to add the Question "What is your gender?"
+    Then I select the "Gender Full" option in the "responseSet" list
+    And I fill in the "search" field with "color"
+    And I click on the button to add the Question "What is your favorite color?"
+    And I click on the "Save" button
+    Then I should not see "Test Form"
```

Comment:
I cant explain how that got there, or why it was passing on my machine, fixed it.